src/main/java/com/alkemy/ong/service/impl/UserServiceImpl.java

```diff
@@ -61,6 +65,12 @@ public UserResponseDto saveUser(RegisterRequest registerRequest) {
 
         newUser.setPassword(passwordEncoder.encode(newUser.getPassword()));
 
+        Role roleUser = rolesRepository.findById(2L).orElseThrow(() -> {
```

Comment:
Hardcoded numbers are known as "Magic Numbers" and are considered a bad practice. Could you please create a constant instead? Ex:  `private static final long USER_ROLE_ID = 2L;`